bin/ledger2beancount-txns

```diff
@@ -15,6 +15,7 @@ use Date::Calc qw(Add_Delta_Days);
 use POSIX qw(ceil);
 use File::BaseDir qw/config_home/;
 use Config::Onion;
+use open qw(:std :utf8);
```

Comment:
does this mean that we're going to assume that input is utf-8?

if so how do we know that is the right thing to do? does, for instance, Ledger-CLI *requires* ledger files to be written in UTF-8?

if so, this approach would be OK (even if depressing :-P)

More generally, the right approach that I'd expect here is that we detect the encoding from locale and rely on it for decoding to unicode code points before any translation.

Reply:
I believe ledger accepts whatever you use, but frankly, it's 2018 and I don't see the point of assuming anything else than UTF-8.